tests/Indexation/AbstractTypeTest.php

```diff
@@ -62,4 +127,9 @@ public function getName(): string
     {
         return 'bar';
     }
+
+    public function getStageForInsert()
```

Comment:
This is useless, it will always return empty array.

Reply:
It is useful actually. FooType extends AbstractType and `$stagedForInsert` can be set via `stageForInsert`

Reply:
Done